README.md

```diff
@@ -57,6 +57,9 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
     - [Logging](#logging)
     - [Localization](#localization)
     - [Machine Learning](#machine-learning)
+        - [Artificial Intelligence](#artificial-intelligence)
+        - [Personal Assistant](#personal-assistant)
+        - [Miscellaneous](#miscellaneous)
```

Comment:
Because of the way these anchor works, it should be Miscellaneous Machine Learning